scripts/cluster_cleanup/README.md

```diff
@@ -0,0 +1,42 @@
+### Zombie Namespace Cleanup Script
+
+This Python script is designed to identify and delete "zombie-" prefixed Kubernetes namespaces and PodMonitors that are older than 12 hours age. It uses the Kubernetes Python client library to connect to the API server and query the list of namespaces and podmonitors and then deleting of them.
+
+### Usage
+
+- Ensure that you have the Kubernetes Python client library installed (pip install -r requirements.txt).
+- Update the prefix and time_delta variables in the script to match your desired settings.
+- Run the script with python cluster_cleanup.py.
+
+### CronJob
+
+To automate the zombie namespace cleanup process, you can use the following Kubernetes CronJob:
+
+```
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cluster-cleanup
+  namespace: cluster-cleanup
+spec:
+  schedule: "0 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: cluster-cleanup
+          containers:
+          - name: cluster-cleanup
+            image: emamihe/cluster-cleanup
```

Comment:
We shouldn't use a personal account as registry for this images, we can use ghcr/docker or other available but with parity or zombienet org.

Reply:
I agree with Javier on this